### DISCUSSION TITLE
Show statistics for Repositories and Orgs

### DISCUSSION THREAD

**Discussion by sachingsachin** [community/community#14892] [Other Feature Feedback, Questions, & Ideas]
It is very difficult to know currently the following very simple statistics and github should show them clearly by default:
1. How many files are there in a repo? (find . -type f)
2. How many lines and words are there per file? (wc -l and wc -m)
3. Total number of lines and words of all files in a repo (Multiply 1 and 2 above)
4. What files are biggest in size?
5. Commit frequency per file and ability to sort on the same.
7. Histogram of commits in a repo
8. What type of files are there in a repo and how many? (Java, HTML, CSS, Ruby etc)
9. First commit and last commit in a repo
10. Number of users per repo
11. Commit frequency per user and ability to sort on the same.
12. Biggest commits per repo in terms of lines/words changed.
13. Commits with large descriptions (These are very useful in finding when an important functionality was changed).

And all of the above should be visible from org view too. So if you go to the org view, currently we just see a list of repos. But it should show a table format with all (or most, as applicable) of the above stats against each repo with the ability to sort. This is super useful in knowing what repos are being most actively worked upon and who are the main committers there.

**Comment by queenofcorgis** [comment]
Thanks for sharing @sachingsachin and the level of detail/thought you put into this list!
